documentation/source/users/arkane/input.rst

```diff
@@ -926,27 +948,12 @@ Energy Grains
 Determine the fineness of the energy grains to be used in the master equation calculations.  Dictate
 the ``maximumGrainSize``, and the ``minimumGrainCount``.
 
+Sensitivity analysis
+--------------------
 
-An example of the algorithm parameters function for the acetyl + O2 network is shown below.
-This example also includes the ``sensitivity_conditions`` attribute which invokes a sensitivity analysis calculation::
-
-    pressureDependence(
-        label='acetyl + O2',
-        Tmin=(300.0,'K'), Tmax=(2000.0,'K'), Tcount=8,
-        Pmin=(0.01,'bar'), Pmax=(100.0,'bar'), Pcount=5,
-        #Tlist = ([300, 400, 600, 800, 1000, 1250, 1500, 1750, 2000],'K')
-        #Plist = ([0.01, 0.1, 1.0, 10.0, 100.0],'bar')
-        maximumGrainSize = (1.0,'kcal/mol'),
-        minimumGrainCount = 250,
-        method = 'modified strong collision',
-        #method = 'reservoir state',
-        #method = 'chemically-significant eigenvalues',
-        interpolationModel = ('chebyshev', 6, 4),
-        #interpolationModel = ('pdeparrhenius'),
-        #activeKRotor = True,
-        activeJRotor = True,
-        sensitivity_conditions = [[(1000, 'K'), (1, 'bar')], [(1500, 'K'), (10, 'bar')]]
-    )
+Cantherm also has the ability to vary barrier heights and generate sensitivity coefficients as a function of changes
```

Comment:
Arkane

Reply:
@goldmanm, can you fix this?

Reply:
Sorry, I didn't see the name change in just the email (so I just squashed the two commits together and thought it was done). Just replaced the two references to Cantherm with Arkane.